### DISCUSSION TITLE
Aint updating

### DISCUSSION THREAD

**Discussion by bryant-the-coder** [ZhiyuanLck/smart-pairs#25] [General]
whenever i try to update the plugins, it shows that failed to update smart-pairs. :( 你知道怎么办吗？

**Comment by ZhiyuanLck** [comment]
Do you try to remove the plugin dir and install again?

**Comment by bryant-the-coder** [comment]
yes it worked xD. Btw u r from? Is there anyway to space out the = sign?

**Reply by ZhiyuanLck** [reply]
What do you mean "space out the = sign"? Can you give an example?

**Reply by bryant-the-coder** [reply]
when i type 
```
local foo _=_ requires "foo"
```

the _ means spaces. It is a feature in autopairs. I liked it as it is aesthetic for the code xD.

**Reply by bryant-the-coder** [reply]
ooh and is it possible to add more info to the readmr? it will be beneficial for newbies

**Reply by bryant-the-coder** [reply]
https://github.com/windwp/nvim-autopairs/wiki/Custom-rules#add-spaces-between-parentheses

you can find the stuff here xD

**Reply by ZhiyuanLck** [reply]
You can create a new issue to request the smart equal feature. And which part of the doc do you think should be detailed more?

**Reply by bryant-the-coder** [reply]
like make a markdown file for advance stuff. Use gifs as people understands more :)

**Reply by ZhiyuanLck** [reply]
No problem about your advice, but they will be arranged after the the main functions are completed.

**Reply by bryant-the-coder** [reply]
> No problem about your advice, but they will be arranged after the the main functions are completed.

anything i can help?

**Reply by ZhiyuanLck** [reply]
If you have some specific idea about the doc or the gif, or any other detailed enhancement, you can just create a pull request.

**Reply by bryant-the-coder** [reply]
> If you have some specific idea about the doc or the gif, or any other detailed enhancement, you can just create a pull request.

ok

**Comment by bryant-the-coder** [comment]
quite interesting that you speak fluent english

**Reply by ZhiyuanLck** [reply]
Not so fluent, it's thanks to the google translater :)
